es6.md

```diff
@@ -189,3 +189,35 @@ interface ImportNamespaceSpecifier {
 ```
 
 A namespace import specifier, e.g., `* as foo` in `import * as foo from "mod.js"`.
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Node {
```

Comment:
@RReverser @ariya @sebmck @impinball thoughts here on what @jeffmo is proposing?